src/main/java/com/scaleunlimited/flinkcrawler/tools/CrawlTool.java

```diff
@@ -55,6 +57,34 @@
         private String _cacheDir;
         private String _commonCrawlId;
         
+        @Option(name = "-agent", usage = "user agent info, format:'name,email,website'", required = false)
+        public void setUserAgent(String agentNameWebsiteEmailString) {
+            if (isCommonCrawl()) {
+                throw new RuntimeException("user agent not used in common crawl mode");
+            }
+            String fields[] = agentNameWebsiteEmailString.split(",", 4);
+            if (fields.length != 3) {
+                throw new RuntimeException("Invalid format for user agent (expected 'name,email,website')");
```

Comment:
Maybe include string that was parsed, so it's clear what was being used. E.g. if they have a space accidentally, then it would be getting processed as another parameter.

Reply:
Done